### ISSUE TITLE
[Bug] stream_to_device doesn't send /api/WebRestoreSession through proxy

### ISSUE THREAD

**Issue by prbsparx** [code42/py42#305]
### Description
Calling `response = sdk.archive.stream_to_device(filePaths, deviceGuid, acceptingDeviceGuid, restorePath, file_size_calc_timeout=0)` leads to a timeout when `settings.proxies` is set.
Specifically, it fails at:
`POST    https://prod-auth-client-04.crashplan.com:4285/api/WebRestoreSession`

### Steps to Reproduce

1. Create a script `RetrieveDeviceBackup.py`:
```
import py42.sdk
import py42.settings as settings
import py42.util as util
import logging

server = "https://www.crashplan.com"
username = "<username>"
password = input("Please input the password for " + username + ": ")

settings.debug.level = logging.DEBUG
settings.proxies = {
	'http': 'http://example.server.com:port',
	'https': 'http://example.server.com:port'
}

sdk = py42.sdk.from_local_account(server, username, password)

filePaths = ["/"]
deviceGuid = "<device GUIDE"
acceptingDeviceGuid = "<accepting"
restorePath = "G:\\path\\<device name>"

response = sdk.archive.stream_to_device(filePaths, deviceGuid, acceptingDeviceGuid, restorePath, file_size_calc_timeout=0)
util.print_response(response)
```
2. Create docker image using dockerfile and tag it py42:1.11.1:
```
FROM python:3.9.2

RUN pip install py42==1.11.1 \
    debugpy
```
3. Run script in Docker image: `docker run -v $(pwd):/app -it py42:1.11.1 python3 app/RetrieveDeviceBackup.py`

### Expected Behavior
Should output the successful results of initiating a device push restore.

### Actual Behavior
times out.

### Basic Information

- py42 version: 1.11.1
- python version: 3.9.2
- operating system: `python:3.9.2` docker image
- forcing traffic to go through a proxy server

**Comment by timabrmsn** [comment]
Hey @prbsparx, thanks for reporting this. I'm taking a look now.

**Comment by prbsparx** [comment]
I saw the bugfix and merge request...
I'm trying to get another computer that needs to be restored to test this on.
Hope to test against py42 commit 3dae923d0d7b8c385b8fb0e0e802efbf932ba6a4 tomorrow.

**Comment by timabrmsn** [comment]
Fixed in release 1.13.0
